tests/jerry/date-tostring.js

```diff
@@ -58,8 +58,9 @@ catch (e)
 
 assert (new Date (NaN).toTimeString () == "Invalid Date");
 assert (new Date (Number.POSITIVE_INFINITY).toString () === "Invalid Date");
-assert (new Date ("2015-02-13").toTimeString () == "00:00:00 GMT+0000");
-assert (new Date ("2015-07-08T11:29:05.023").toTimeString () == "11:29:05 GMT+0000");
+assert (new Date ("2015-02-13").toGMTString () == "Fri, 13 Feb 2015 00:00:00 GMT");
+assert (new Date ("2015-07-08T11:29:05.023+01:00").toGMTString () == "Wed, 08 Jul 2015 10:29:05 GMT");
+assert (new Date ("2015-07-08T11:29:05.023+10:00").toGMTString () == "Wed, 08 Jul 2015 01:29:05 GMT");
```

Comment:
Does this worth in seperate PR?. Seems not relate to jerry-port

Reply:
It's somewhat related, as the prototype of the port api function for local tza has been changed, which required the implementation for it to be updated. This in turn fixed a former issue, which caused incorrect TZA handling in some cases.
These tests were testing the former incorrect behavior and needed to be updated.